salt/states/zabbix_host.py

```diff
@@ -21,6 +21,107 @@ def __virtual__():
         return True
     return (False, "zabbix module could not be loaded")
 
+def _interface_format(interfaces_data):
+    """
+    Formats interfaces from SLS file into valid JSON usable for zabbix API.
+    Completes JSON with default values.
+
+    :param interfaces_data: list of interfaces data from SLS file
+
+    """
```

Comment:
Thank you for pointing me in the right direction @piterpunk. I will move and re-arrange things soon. Some other projects have taken my priority and so this PR will have to wait a bit, but I do intend to finish it so that the fixes can get merged.

Reply:
No problems, @rsolano-isb , I guess everyone understands the problem of too little time for too much things to do. Probably most of us are also affected by it.